src/plays/tic-tac-toe/TicTacToe.css

```diff
@@ -0,0 +1,36 @@
+.icon {
+  color: red;
+  font-size: 30px;
+}
+h1 {
+  color: black;
+  padding: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+h3 {

```

Comment:
Do not override selectors directly. Scope it to your play with a related class name.